packages/utils/scaffold.ts

```diff
@@ -33,52 +31,52 @@ function mapOptionsToTransform(config) {
  * and writes the file
  */
 
-module.exports = function runTransform(webpackProperties, action) {
+export default function runTransform(webpackProperties: IConfig, action: string) {
```

Comment:
Iconfig is probably not a good idea to name this, name it something related to a `scaffold-interface`.

Reply:
The same object has different names. We called it `config` in modify-config-helper and then the same object is called `webpackProperties` here. I changed both to `transformConfig`. I think that suits better and consistent.